mautic/mautic
# Add dynamic content to the Email Builder in MJML mode

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          |  no
| New feature/enhancement? (use the a.x branch)      |  yes
| Deprecations?                          |  no
| BC breaks? (use the c.x branch)        |  no
| Automated tests included?              |  no
| Related user documentation PR URL      | no
| Related developer documentation PR URL | in the code
| Issue(s) addressed                     | Fixes #10240


#### Description:

Makes the existing Dynamic Content block available in MJML. 

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new MJML E-Mail (e.g. Brienz theme)
3. Drag the Dynamic Content block to the canvas and configure it
4. Save and Close
5. Preview the E-Mail in the HTML Preview: Default content should be shown
6. Send a Test Email: Default content should be shown
7. Send a real segment or campaign email and check in the E-Mail client to see if the parameters worked as expected. Either the default or the variant should be shown




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10782"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
